pkg/cli/mirror/additional.go

```diff
@@ -47,20 +47,20 @@ func (o *AdditionalOptions) Plan(ctx context.Context, imageList []v1alpha2.Image
 
 		ref := srcRef.Ref.Exact()
 		if !image.IsImagePinned(ref) {
-			srcImage, err := image.ResolveToPin(ctx, resolver, ref)
```

Comment:
This will have to be revisited. Looks like it ended up getting a lot more changes than logging.